cpu/nrf52/periph/usbdev.c

```diff
@@ -0,0 +1,654 @@
+/*
+ * Copyright (C) 2018 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup sam0_common
+ * @{
+ * @file
+ * @brief   USB interface functions
+ *
+ * @author  Koen Zandberg <koen@bergzand.net>
+ * @}
+ */
+#include <stdint.h>
+#include <stdlib.h>
+#include <errno.h>
```

Comment:
I didn't see any errno, maybe this can be removed ?

Reply:
Required for the `-ENOTSUP` return values